codeception.yml

```diff
@@ -1,17 +1,24 @@
-namespace: UuidBehavior
```

Comment:
Since `spryker/uuid-behavior` is a separate from `spryker/spryker` repo, I assume, you should keep old (`UuidBehavior`).

Please take a look at EventBehavior, for example:
- https://github.com/spryker/event-behavior/blob/master/codeception.yml

Reply:
I think actually @odokienko 's solution is the correct one.